.eslintrc.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
     'prettier/react',
     'prettier/@typescript-eslint',
   ],
-  plugins: ['@typescript-eslint', 'prettier'],
+  plugins: ['@typescript-eslint', 'prettier', 'react-hooks'],
```

Comment:
Do we need to enable the `react-hooks` plugin? I think `react-app` in `extends` takes care of enabling the hooks lint rules.

Reply:
ya doesn't apparently need this